requirements.txt

```diff
@@ -0,0 +1,4 @@
+pyyaml
+-e git+https://github.com/edx/XBlock.git@tag-master-2015-05-22#egg=XBlock
+-e git+https://github.com/edx/xblock-utils.git@b4f9b51146c7fafa12f41d54af752b8f1516dffd#egg=xblock-utils
```

Comment:
Why the `-e` flags? I'm pretty sure that we don't want these in requirements files. (edx-platform has a lot of these `-e` dependencies, and they've caused us a fair amount of trouble in the past -- we're trying to get away from that.)